backend/src/test/java/com/bakdata/conquery/integration/tests/RestartTest.java

```diff
@@ -148,6 +148,8 @@ public void execute(String name, TestConquery testConquery) throws Exception {
 
 		test.executeTest(support);
 
+		storage = support.getMetaStorage();
+
```

Comment:
wird das noch gebraucht?

Reply:
In der Datei wurde glaube ich nur `storage = support.getMetaStorage();` zugefügt. Keine andere Änderung  - sah komisch aus.

Aber kann gut sein, dass nach dem Neustart der Storage neu assgined werden muss.